validation/report.md

```diff
@@ -134,6 +162,18 @@ Figure 7
 
 
+## External Validation
+
+### Edge
+Google Maps Satellite View was used to understand what exists in spaces that contains Official edges, but not OSM edges. For each city, one example was examined. In all three examples, the edges should not be included in the analysis, as they do not represent paths that are a part of network circulation. 
+ 
+In Bern, the space examined exist on the Northeastern area of the study area. When examined on Google Maps, the space appears to be an industrial facility. In Hong Kong, the space examined exist close to the center of the study area. When examined on Google Maps, the space appears to be an railroad juntion. In Olomouc, the space examined exist on the Eastern edge of the study area. When examined on Google Maps, the space appears to be an industrial facility. In all three examples, the edges included in the Official dataset, but not the OSM derived dataset, are access roads that are for private establishments. These edges should not be included in the analysis, and this further proves that the OSM dataset is more robust. 
+
+
+### Destination
+In both cities, 10 OSM derived destinations and Official destinations were chosen at random. Each of the 40 destinations were then inspected based on their latitudes and longitudes on Google Maps Satellite View, Google Street Maps, and Yelp. Of the 20 OSM derived destinations, 18 are confirmed to be food vendors. Both of the false destinations occured in Olomouc and are pharmacies, but one of them still has a Google Tag as being a bakery with photos, showing that the space was recently converted into a pharmacy. Of the 20 Official destinations, 12 are confirmed food destinations. In the 8 false destinations, 4 occured in each city. In Olomouc, one false destination is a construction site, and the other three are other retail spaces. In Sao Paolo, one false destination was located on a single family home, and a different false destination was located in the middle of a large intersection. The ground truthing examples show that the Official data is not completely accurate, and is possibly even less accurate than the OSM derived data. This, along with the consistent data format present in OSM derived data, further supports the use of OSM derived data.  
```

Comment:
Great! A few suggestions...

  - 10 for OSM and 10 more for official in each city... state that clearer when introduced here
  - Can you do 10 more in each city/dataset? i.e., double the current analysis?
  - Random how? Explain sampling strategy here. If using numpy/pandas random selection, make sure you're seeding numpy with an integer so that the precise randomization is reproducible.
  - Can you provide any info about dates from the Google data? Are any dates available? Like, the Google imagery is copyrighted as-of when... this is important to get a sense of which is out of date. Maybe Google is 3 years old, and official is brand new! Unlikely, but we want some such evidence to bolster our claim.
  - In SP, 2 of the 4 falses are accounted for... what's the story with the others?
  - for the one in the middle of a large intersection, was there a reasonable matching destination along one of those intersecting streets?